src/generated/.openapi-generator/FILES

```diff
@@ -128,134 +127,3 @@ models\WardDTO.ts
 models\index.ts
 runtime.ts
 tsconfig.json
-=======
```

Comment:
Thanks 🙏

Reply:
Is there something to resolve here ?

Reply:
No , I am just thanking you for fixing this unrelated issue 😊